lib/brains/tests/test_forms.py

```diff
@@ -131,12 +134,32 @@ def test_ok(self):
 
     def test_ok_method(self):
         seller, seller_product = create_seller()
+        self.product_mock.get('moz-brick').recurrence = None
         method = create_method(create_braintree_buyer()[1])
+
         form, errors = self.process({
-            'amount': '5',
+            'amount': '10.00',
             'paymethod': method.get_uri(),
             'product_id': seller_product.public_id
         })
         assert not errors, errors.as_text()
         assert 'payment_method_nonce' not in form.braintree_data
         eq_(form.braintree_data['payment_method_token'], method.provider_id)
+
+    def test_recurring(self):
+        seller, seller_product = create_seller()
```

Comment:
this would be less confusing to me if you explicitly added the line `seller_product.recurrence = 'monthly'`